.github/workflows/target-creation.yml

```diff
@@ -0,0 +1,41 @@
+name: CI - Upload config.h to API
+
+on:
+  push:
+    branches:
+    - 'master'
+
+jobs:
+  deploy:
+    name: Create or Update Target
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - id: files
+      uses: tj-actions/changed-files@v34
+      with:
+        files: |
+          src/config/**
+
+    - if: steps.files.outputs.any_changed == 'true'
```

Comment:
Then this if will not be needed, but I suppose it does not hurt.

Reply:
I think I would still like to keep this check in - it doesn't hurt.